Makefile

```diff
@@ -8,6 +8,7 @@ generate:
 	find pkg/ -name "*generated*.go" -exec rm {} -f \;
 	./hack/build-go.sh generate ${WHAT}
 	goimports -w -local kubevirt.io cmd/ pkg/ tests/
+	./hack/bootstrap-ginkgo.sh test ${WHAT}
```

Comment:
You've spotted an evil copy-paste. :)

It's only supposed to generate the structure, not run the tests. Will fix the tests.